transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -220,6 +243,15 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
           if (isFileMetadata) transformFileMetadata(entityType, filename, metadata)
           else transformMetadata(entityType, filename, metadata)
       }
+    // generate a file ingest request (if applicable)
+    if (isFileMetadata) {
+      val fileIngestRequests = processedData.map(generateFileIngestRequest(_, inputPrefix))
+      StorageIO.writeJsonLists(
+        fileIngestRequests,
+        entityType,
+        s"${outputPrefix}/data/${entityType}"
```

Comment:
I'd use something other than `data`, since that already has a meaning in the common layout. Maybe `data-transfer-requests`?

Reply:
Sounds good! I just updated it.